### PR TITLE
Bug 1155582 - Sync the phone and email details fragments used when displaying a contact in the call log information pane r=drs,francisco

### PR DESCRIPTION
**Pull request by gabrielesvelto**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=61cddd3c78db4755063a4c4b6da4679f78581d9a

File: apps/communications/dialer/locales/dialer.en-US.properties [hunk 1]
```diff
@@ -107,6 +107,13 @@ delete-n-log?[few]   = Delete {{ n }} logs?
 delete-n-log?[many]  = Delete {{ n }} logs?
 delete-n-log?[other] = Delete {{ n }} logs?
 
+# Call log information
+composeEmail.ariaLabel    = Email
+emailDetail.ariaLabel     = Email
+phoneDetail.ariaLabel     = Phone
+send-sms-button.ariaLabel = SMS
+separator                 = ,
```
Comment (arcturus) [inline apps/communications/dialer/locales/dialer.en-US.properties]: Are we using this id?

Comment (gabrielesvelto) [inline apps/communications/dialer/locales/dialer.en-US.properties]: Yes, see here: https://github.com/gabrielesvelto/gaia/blob/bug-1155582-sync-phone-email-contact-details/apps/communications/contacts/elements/phone_details.html#L8

Comment (arcturus) [inline apps/communications/dialer/locales/dialer.en-US.properties]: Perfect, thanks!

**Pull request merged**